src/components/course/uc/deleteUserData/deleteUserDataSteps.js

```diff
@@ -18,7 +19,7 @@ const addLessonContentsToTrashbinData = (userId, lessons = [], trashBinData) =>
 };
 
 const validateParams = (userId) => {
-	if (!isValidObjectId(userId)) throw new ValidationError('a valid objectId is required', { userId });
+	if (!isValidObjectId(userId)) throw new AssertionError(missingParameters({ userId }));
```

Comment:
Should'nt we use a dedicated helper here, e.g. `invalidObjectId()`?

Reply:
good point, until now i just edited the existing helper, for sure we should use it here. However I was just taking care in that single commit to not use validation error with wrong (string) constructor.